src/viewport.js

```diff
@@ -936,40 +930,79 @@ $.Viewport.prototype = /** @lends OpenSeadragon.Viewport.prototype */{
         return changed;
     },
 
-
     /**
-     * Convert a delta (translation vector) from pixels coordinates to viewport coordinates
-     * @function
-     * @param {Boolean} current - Pass true for the current location; defaults to false (target location).
+     * Scale a delta (translation vector) from pixels coordinates to viewport
+     * coordinates. This method does not take rotation into account.
+     * Consider using deltaPixelsFromPoints if you need to account for rotation.
+     * @param {OpenSeadragon.Point} deltaPoints - The translation vector to convert.
+     * @param {Boolean} [current=false] - Pass true for the current location;
+     * defaults to false (target location).
+     * @returns {OpenSeadragon.Point}
      */
-    deltaPixelsFromPoints: function( deltaPoints, current ) {
+    scaleDeltaPixelsFromPoints: function(deltaPoints, current) {
```

Comment:
They are both needed.
deltaPointsFromPixels: https://github.com/openseadragon/openseadragon/blob/master/src/viewer.js#L2292
scaleDeltaPixelsFromPoints: https://github.com/openseadragon/openseadragon/pull/796/files#diff-80bee396a520b6fa750064722bc3f0ceR655

Maybe `scaleDeltaPointsFromPixels` and `deltaPixelsFromPoints` are not used right now but it seems coherent to have them too.